index.html

```diff
@@ -1443,7 +1443,7 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\serializedMethodData]]</dfn>
+              <dfn data-export="">[[\serializedMethodData]]</dfn>
```

Comment:
```suggestion
              <dfn data-export="" data-dfn-for="PaymentRequest">[[\serializedMethodData]]</dfn>
```

Reply:
I'm not sure if all of these need to be exported or not... are they all accessed by the basic card spec?

Reply:
@marcoscaceres yes, they are. the issue I raised [here](https://github.com/w3c/payment-request/issues/890) links to the parts of basic card spec that link to these terms. I double-checked by searching for `data-cite` in [this branch](https://github.com/w3c/payment-method-basic-card/pull/84) to make sure they're still there.

Reply:
ok, thanks for confirming.